src/javascript/binary/websocket_pages/user/account/authenticate.js

```diff
@@ -133,8 +133,7 @@ const Authenticate = (() => {
             $files.each((i, e) => {
                 if (e.files && e.files.length) {
                     const $e = $(e);
-                    const id = $e.attr('id');
-                    const type = `${($e.attr('data-type') || '').replace(/\s/g, '_').toLowerCase()}_${id.split('_')[0]}`;
+                    const type = `${($e.attr('data-type') || '').replace(/\s/g, '_').toLowerCase()}`;
```

Comment:
none of the `data-type` values have any space, is this just for forward-proofing?